sbt

```diff
@@ -130,10 +133,10 @@ make_url () {
 
   case "$version" in
         0.7.*) echo "$base/files/sbt-launch-0.7.7.jar" ;;
-      0.10.* ) echo "$base/org.scala-tools.sbt/sbt-launch/$version/sbt-launch.jar" ;;
-    0.11.[12]) echo "$base/org.scala-tools.sbt/sbt-launch/$version/sbt-launch.jar" ;;
-          0.*) echo "$base/org.scala-sbt/sbt-launch/$version/sbt-launch.jar" ;;
-            *) echo "$base/org/scala-sbt/sbt-launch/$version/sbt-launch.jar" ;;
+      0.10.* ) echo "$base/org.scala-tools.sbt/sbt-launch/$version/sbt-launch-${version}.jar" ;;
+    0.11.[12]) echo "$base/org.scala-tools.sbt/sbt-launch/$version/sbt-launch-${version}.jar" ;;
+          0.*) echo "$base/org.scala-sbt/sbt-launch/$version/sbt-launch-${version}.jar" ;;
+            *) echo "$base/org/scala-sbt/sbt-launch/$version/sbt-launch-${version}.jar" ;;
```

Comment:
This change will break the <1.0.0 jars.

What is the motivation for this change?

Reply:
The JARs without version number do not have signatures.
The ones with version number are not identical to the ones without (I assume they are recompiled from the same code), so we cannot use the versioned signature to verify the unversioned JAR.

Reply:
I see!

Let's push this change to later when we consider adding signature verification.

```suggestion
            *) echo "$base/org/scala-sbt/sbt-launch/$version/sbt-launch.jar" ;;
```

Reply:
You can't the same is true for the files with the md5 hashes

Reply:
Hmm.  Well, that's concerning.

If you don't mind, I'm going to keep this subthread open as I want to understand why or, more importantly, how they're different.

Reply:
I actually jumped on this straight away.  Using https://github.com/scala/jardiff there's no difference.  From vimdiff it looks like it's just timestamps.  So I guess we're fine to make this change now.